### PR TITLE
moveit_grasps tutorial

### PR DESCRIPTION
**Pull request by henningkayser**
This will be the new tutorial for moveit_grasps.
Currently, there is only the kinetic branch so I will update all links and references once melodic (or master) branch are created.
Should I also target this PR against melodic and kinetic?
@davetcoleman I addressed you previous changes.

Also, I'm currently testing moveit_grasps and will add some instructions to the tutorial if needed.

### CODE REVIEW

**Comment by rhaschke** [comment]
How does this relate to https://github.com/ros-planning/moveit_tutorials/pull/215?

**Comment by henningkayser** [comment]
> How does this relate to #215?

Good question! @davetcoleman do you think the previous PR should be continued rather than making a new one?

File: doc/moveit_grasps/moveit_grasps_tutorial.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+To load that file at launch, you copy the example in the file `launch/load_panda.launch <https://github.com/ros-planning/moveit_grasps/blob/kinetic-devel/launch/load_panda.launch>`_ where you should see the line::
+
+    ...
+    
+    <rosparam command="load" file="$(find moveit_grasps)/config_robot/panda_grasp_data.yaml"/>
+    
+    ...
+
+Within that file you will find all of the gripper specific parameters necessary for customizing MoveIt Grasps with any suction or finger gripper.
```
Comment (rhaschke) [inline doc/moveit_grasps/moveit_grasps_tutorial.rst]: From looking at the parameter file, the meaning of all those parameters doesn't yet become clear.
I guess the code is limited to parallel-jaw grippers, right? Why, for example, is it required to define the finger width?

**Comment by mlautman** [comment]
@henningkayser I have made some additional changes that I have pushed to the picknikrobotics moveit-tutorials fork. Please consider cherry picking them on top of your branch.

https://github.com/PickNikRobotics/moveit_tutorials/commits/melodic-mike-moveit-grasps

My changes are in the two most recent commits

**Comment by henningkayser** [comment]
@mlautman Looks good! You could have pushed them here directly as well. Shouldn't we copy the remaining images into this repo?

File: doc/moveit_grasps/moveit_grasps_tutorial.rst (hunk 1 above)
Comment (henningkayser) [reply doc/moveit_grasps/moveit_grasps_tutorial.rst]: I added further parameter description to the example config https://github.com/ros-planning/moveit_grasps/pull/36. I agree that some of the parameters seem a bit confusing at first but I think the comments should clarify all of them.

**Comment by henningkayser** [comment]
> How does this relate to #215?

@rhaschke actually this was a duplicate since I wasn't aware that there was already an existing PR for moveit_grasps. Also the other PR was abandoned and incomplete so I would vote for merging this one and closing the other.

**Pull request merged**